webapp-django/crashstats/crashstats/jinja2/crashstats_base.html

```diff
@@ -81,7 +81,7 @@ <h1>Product Navigation</h1>
           <li class="version_select">
             <label for="product_version_select" class="visually-hidden">Select Version:</label>
             <select id="product_version_select" >
-              <optgroup label="Active">
+              <optgroup label="Featured">
```

Comment:
I spent some time fixing some cases of "active" which should have been "featured".